subprojects/docs/src/docs/userguide/migrating_from_maven.adoc

```diff
@@ -77,50 +85,200 @@ $ gradle build
+The appropriate configuration in the Gradle build looks like this:
++
 [source,groovy]
 ----
-enableFeaturePreview("IMPROVED_POM_SUPPORT")
+dependencies {
+    testCompile 'org.codehaus.groovy:groovy-all:2.5.4'
+}
 ----
++
+The result of this will be that all `compile` and `runtime` scope dependencies in the `groovy-all` POM get added to the test runtime classpath, while only the `compile` scope dependencies get added to the test compilation classpath.
+Dependencies with other scopes will be ignored.
+
+[[migmvn:declaring_repos]]
+=== Declaring repositories
+
+Gradle allows you to retrieve declared dependencies from any Maven-compatible or Ivy-compatible repository.
+Unlike Maven, it has no default repository, so you have to declare at least one.
+In order to have the same behavior as your Maven build, just configure <<repository_types#sub:maven_central,Maven Central>> in your Gradle build, like this:
+
+[source,groovy]
+----
+repositories {
+    mavenCentral()
+}
+----
+
+You can also use the `repositories {}` block to configure custom repositories, as described in the <<repository_types#sub:maven_repo,Repository Types>> chapter.
 
-The BOM support in Gradle works similarly to using `<scope>import</scope>` when depending on a BOM in Maven.
-In Gradle however, it is done via a regular dependency declaration on the BOM.
+[[migmvn:excluding_deps]]
+=== Excluding transitive dependencies
+
+Maven builds use exclusions to keep unwanted dependencies -- or unwanted _versions_ of dependencies -- out of the dependency graph.
+These can be tricky to migrate not because Gradle doesn't have good support for such exclusions, but because you really need to understand _why_ an exclusion is in place to migrate it properly.
+
+If you want to exclude a dependency for reasons unrelated to versions, then check out the section on <<managing_transitive_dependencies#sec:excluding_transitive_module_dependencies,Excluding transitive module dependencies>>.
+It shows you how to attach an exclusion either to an entire configuration (often the most appropriate solution) or to a dependency.
+You can even easily apply an exclusion to all configurations.
+
+If you're more interested in controlling which version of a dependency is actually resolved, then Gradle has better options than exclusions.
+Consider using <<managing_transitive_dependencies#sec:dependency_constraints,dependency constraints>> or <<managing_transitive_dependencies#sec:enforcing_dependency_version,forcing a particular version>>.
+
+[[migmvn:optional_deps]]
+=== Handling optional dependencies
+
+There are two sides to optional dependencies: 
+
+ * How the build treats them as transitive dependencies
+ * How declared dependencies are published as optional
+
+For the first scenario, Gradle simply ignores any transitive dependencies that are declared as optional.
+They are not resolved and have no impact on the versions selected if the same dependencies appear elsewhere in the dependency graph as non-optional.
```

Comment:
`...Gradle simply ignores...` -> `...Gradle does what Maven does: it simply ignores...`